parm/land/obs/config/adpsfc_snow.yaml

```diff
@@ -0,0 +1,97 @@
+obs space:
+  name: adpsfc_snow
+  distribution:
+    name: Halo
+    halo size: 250e3
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: $(OBS_DIR)/adpsfc_snow_$(OBS_DATE).nc4
```

Comment:
I think we probably want to follow a convention more akin to what is in ops now, suggest:
`$(OBS_DIR)/$(OBS_PREFIX)adpsfc_snow.$(OBS_DATE).nc4`

So that it can be `gdas` and `gfs` if we want.

Reply:
Added $(OBS_PREFIX). Thanks.

Reply:
@CoryMartin-NOAA When would something be labeled GFS and when would it be GDAS? Are there any other options?

Reply:
GFS = early cycle 
GDAS = late cycle

That should be the only options for global, it helps distinguish which obs are available (latency, etc.)